src/applications/mhv-medical-records/tests/components/DownloadFileType.unit.spec.jsx

```diff
@@ -225,10 +225,10 @@ describe('DownloadFileType — AAL logging', () => {
     fireEvent.click(btn);
   };
 
-  it('logs AAL success when PDF download succeeds', async () => {
+  it.skip('logs AAL success when PDF download succeeds', async () => {
     const screen = renderWithFormat('pdf');
     await clickDownload(screen);
-
+    // TODO: This feature is not in production yet. It are only testing it in stage.
```

Comment:
QUESTION: What does production status have to do with e2e tests? If the feature is built we should running the tests, especially if we are going to be releasing this feature to production.